src/Platforms/MariaDb1070Platform.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Doctrine\DBAL\Platforms;
+
+use Doctrine\DBAL\Types\Types;
+
+/**
+ * Provides the behavior, features and SQL dialect of the MariaDB 10.2 (10.2.7 GA) database platform.
+ *
+ * Note: Should not be used with versions prior to 10.2.7.
+ */
```

Comment:
This doc block looks like copy&paste from the 10.2.7 platform. It should be adjusted to match what this new class actually represents.